semantic/src/site/globals/site.overrides

```diff
@@ -1,3 +1,38 @@
 /*******************************
          Site Overrides
 *******************************/
+
+p {
+  font-size: 15px;
+  letter-spacing: 0;
+}
+
+.ui.basebody.container {
+  background: rgb(248, 248, 248);
+}
+
+.ui.basic.buttons .button, .ui.basic.button  {
+                color: #003BFE
+}
+
+.sup .ui.tiny.progress {
```

Comment:
oops, my play around class names are not very professional `sup` and `dreistripes` lol, would be good to change to something more appropriate

Reply:
although stripes are probably not used in this pr

Reply:
Yes, I just moved the whole file to here - I'll do another PR for changes to the list (hopefully) today and will branch from this one so will keep this here but renamed.